### PR TITLE
test configs folder cleanup

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

**Comment by rukai** [comment]
Thanks for taking this on!

File: shotover-proxy/tests/redis_int_tests/mod.rs [hunk 1]
```diff
@@ -130,27 +130,27 @@ async fn tls_source_and_tls_single_sink() {
...
+    //     let shotover =
+    //         shotover_process("tests/test-configs/redis/tls-no-verify-hostname/topology.yaml")
+    //             .start()
+    //             .await;
+    //
+    //     let mut connection = redis_connection::new_async_tls("127.0.0.1", 6379).await;
+    //     test_cluster_basics(&mut connection).await;
+    //     shotover.shutdown_and_then_consume_events(&[]).await;
+    // }
 }
```
Comment (rukai) [inline shotover-proxy/tests/redis_int_tests/mod.rs:147]: Why is this commented out?

Comment (conorbros) [reply shotover-proxy/tests/redis_int_tests/mod.rs:147]: Oops

**Review by rukai — APPROVED** [review]
looks good, lets land this!

**Comment by rukai** [comment]
Context for the next reviewer:
We started putting our new test configs in per protocol folders recently.
This pr changes our old test configs to follow this new format.

**Review by cjrolo — APPROVED** [review]

**Pull request merged**